runlog.c

```diff
@@ -312,6 +312,7 @@ run_add_record(
         int            eoln_type,
         int            variant,
         int            is_hidden,
+        int            upsolving,
```

Comment:
This parameter is not necessary. The 'upsolving' value is not stored, but only used for parameter check.